### PR TITLE
Add initial post for JuMP-dev 2025

### PR DESCRIPTION
**Pull request by odow**
Tagging a bunch of people. No pressure or expectation, but if you would like to join the program committee, please comment with your current affiliation.

cc @mlubin @blegat @joaquimg @jd-foster @pulsipher @bernalde @tjdiamandis @azzaare @lkapelevich @chriscoey @frapac @matbesancon

### CODE REVIEW

**Comment by lkapelevich** [comment]
I would be happy to join. I am at Amazon.

**Comment by jd-foster** [comment]
Yes, happy to join. CSIRO

**Comment by OlegBarbin** [comment]
Happy to help, I'm at Fonterra.

**Comment by odow** [comment]
We have the makings of a very antipodean organizing committee!

**Comment by joaquimg** [comment]
So I will crash the oceanic party, but keep it below the equator! PSR

**Comment by blegat** [comment]
I'd be happy to join as well, UCLouvain

**Comment by matbesancon** [comment]
happy to help this year (Inria, Université Grenoble Alpes)

File: _posts/2024-09-26-jump_dev_2025.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+If you have an idea for a talk and are not sure if it will be of interest, ask
+the program committee by writing to `jump-dev-2025@googlegroups.com`.
+
+See the previous workshops for examples of accepted talks:
+[2017](/meetings/mit2017),
+[2018](/meetings/bordeaux2018),
+[2019](/meetings/santiago2019),
+[2021](/meetings/juliacon2021),
+[2022](/meetings/juliacon2022),
+[2023](/meetings/juliacon2023).
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:76]:
```suggestion
[2023](/meetings/juliacon2023), and
```

File: _posts/2024-09-26-jump_dev_2025.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+JuMP-dev will be held at the [GridAKL](https://gridakl.com) in downtown
+Auckland, New Zealand. There are a wide range of accommodation options within
+walking distance.
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1900.7256269489792!2d174.75809299292328!3d-36.84403770139611!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d0d47f420b07ccd%3A0x754b09a7cddaa7cf!2sGrid%20Auckland%20%2F%20John%20Lysaght%20Coworking%20Startup%20space!5e0!3m2!1sen!2snz!4v1727393894842!5m2!1sen!2snz" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+
+## Program committee
+
+ * Oscar Dowson (Program chair; local organization)
+ * Oleg Barbin (Fonterra; local organization)
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:90]:
```suggestion
 * Oleg Barbin (Fonterra; local organization)
 * Mathieu Besançon (Inria, Université Grenoble Alpes)
```

File: _posts/2024-09-26-jump_dev_2025.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon at GridAKL
+
+## Key dates
+
+ * September 27 2025: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
+ * February 28 2025: talk submissions close
+ * March 31 2025: talk accept notifications
+ * November 17--20 2025: JuMP-dev 2025
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:40]: Program committee: thoughts on these dates?

**Comment by mlubin** [comment]
Hudson River Trading :)

**Comment by bernalde** [comment]
Cannot commit to travel, but happy to help in the program committee.
My affiliation is Purdue University (Davidson School of Chemical Engineering)

**Comment by Azzaare** [comment]
Happy to join the crew from Internet Initiative Japan

File: _posts/2024-09-26-jump_dev_2025.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+JuMP-dev will be held at the [GridAKL](https://gridakl.com) in downtown
+Auckland, New Zealand. There are a wide range of accommodation options within
+walking distance.
+
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1900.7256269489792!2d174.75809299292328!3d-36.84403770139611!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d0d47f420b07ccd%3A0x754b09a7cddaa7cf!2sGrid%20Auckland%20%2F%20John%20Lysaght%20Coworking%20Startup%20space!5e0!3m2!1sen!2snz!4v1727393894842!5m2!1sen!2snz" width="600" height="450" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
+
+## Program committee
+
+ * Oscar Dowson (Program chair; local organization)
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:89]:
```suggestion
 * Oscar Dowson (Program chair; local organization)
 * Jean-François Baffier (Internet Initiative Japan)
```

**Comment by odow** [comment]
Okay. I'm closing volunteer signups. 10 should be enough :smile:

Committee: please take a read of the post and let me know if you think there should be any changes.

File: _posts/2024-09-26-jump_dev_2025.md (hunk 3 above)
Comment (OlegBarbin) [reply _posts/2024-09-26-jump_dev_2025.md:40]: I'm assuming talk submissions open should be September 27 2024 instead?

File: _posts/2024-09-26-jump_dev_2025.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+excursion. Schedule is subject to change at any time.
+
+ * Monday 17 November: talks at GridAKL
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon at GridAKL
+
+## Key dates
+
+ * September 27 2025: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:37]:
```suggestion
 * September 27 2024: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
```

File: _posts/2024-09-26-jump_dev_2025.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+ * Monday 17 November: talks at GridAKL
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon at GridAKL
+
+## Key dates
+
+ * September 27 2024: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
+ * February 28 2025: talk submissions close
+ * March 31 2025: talk accept notifications
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:39]:
```suggestion
 * March 31 2025: talk acceptance notifications
```

**Comment by jd-foster** [comment]
It reads well - looks good to me.

File: _posts/2024-09-26-jump_dev_2025.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+## Schedule
+
+The workshop will be structured as two days of talks, a hackathon, and a local
+excursion. Schedule is subject to change at any time.
+
+ * Monday 17 November: talks at GridAKL
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon at GridAKL
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:33]:
```suggestion
 * Thursday 20 November: hackathon (venue to be confirmed)
```

File: _posts/2024-09-26-jump_dev_2025.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+excursion. Schedule is subject to change at any time.
+
+ * Monday 17 November: talks at GridAKL
+ * Tuesday 18 November: talks at GridAKL
+ * Wednesday 19 November: excursion
+ * Thursday 20 November: hackathon (venue to be confirmed)
+
+## Key dates
+
+ * September 27 2024: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
```
Comment (odow) [inline _posts/2024-09-26-jump_dev_2025.md:37]:
```suggestion
 * October 2024: [talk submissions open](https://forms.gle/SzbqwzfsRNW97PZt5)
```

File: _posts/2024-09-26-jump_dev_2025.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+development team.
+
+All participants will uphold the [JuMP Code of Conduct](https://github.com/jump-dev/JuMP.jl/blob/master/CODE_OF_CONDUCT.md).
+
+## Schedule
+
+The workshop will be structured as two days of talks, a hackathon, and a local
+excursion. Schedule is subject to change at any time.
+
+ * Monday 17 November: talks at GridAKL
```
Comment (bernalde) [inline _posts/2024-09-26-jump_dev_2025.md:30]: Might be worth to define what GridAKL is

Comment (odow) [reply _posts/2024-09-26-jump_dev_2025.md]: Added a link to the Venue section

**Pull request merged**